floodsub.go

```diff
@@ -414,8 +418,10 @@ func (p *PubSub) GetTopics() []string {
 
 // Publish publishes data under the given topic
 func (p *PubSub) Publish(topic string, data []byte) error {
-	seqno := make([]byte, 8)
-	binary.BigEndian.PutUint64(seqno, uint64(time.Now().UnixNano()))
+	seqno := make([]byte, 16)
+	counter := atomic.AddUint64(&p.counter, 1)
+	binary.BigEndian.PutUint64(seqno[:8], uint64(time.Now().UnixNano()))
```

Comment:
If we initially set `counter` to `time.Now().UnixNano()`, then we don't need to get the current time every time we send a message. 

Not a big deal, but might save us some syscalls

Reply:
It also saves us from having to send a whole extra 8 bytes per message

Reply:
good points, nice small efficiencies. i'll open a new PR.

Reply:
actually, there is no syscall involved in getting the time; but still.

Reply:
opened follow-up PR.